Dockerfile.java-alpine

```diff
@@ -77,12 +77,30 @@ RUN mkdir /app && \
     chown -R 10001:0 /app/volumes && \
     find /app \( -type d -exec chmod ug+rwx {} \; -exec chown 10001:0 {} \; -o -type f -exec chmod ug+rw {} \; -exec chown 10001:0 {} \; \)
 
+ENV JAVA_OPTIONS=
+ENV QUBERSHIP_PROFILER_ENABLE=N
+ARG QUBERSHIP_PROFILER_VERSION=1.0.4
+ARG MAVEN_CENTRAL_URL=https://repo.maven.apache.org/maven2
+
+RUN mkdir /app/execution-statistics-collector && \
+    cd /app/execution-statistics-collector && \
+    # curl -OL $MAVEN_CENTRAL_URL/org/qubership/profiler/qubership-profiler-installer/$QUBERSHIP_PROFILER_VERSION/qubership-profiler-installer-$QUBERSHIP_PROFILER_VERSION.zip && \
+    unzip qubership-profiler-installer-$QUBERSHIP_PROFILER_VERSION.zip && \
+    rm qubership-profiler-installer-$QUBERSHIP_PROFILER_VERSION.zip && \
+    chown -R 10001:0 /app/execution-statistics-collector
+
 VOLUME /tmp
 VOLUME /etc/env
 VOLUME /app/nss
 VOLUME /etc/ssl/certs/java
 VOLUME /etc/secret
 
+RUN mkdir /app/execution-statistics-collector/dump && \
+    chmod ug+rw -R /app/execution-statistics-collector && \
+    chown -R 10001:0 /app/execution-statistics-collector
+
+VOLUME /app/execution-statistics-collector/dump
```

Comment:
Any recommendations on the folder name when storing the profile results locally?